tests/ci/ci.py

```diff
@@ -2121,6 +2121,8 @@ def main() -> int:
                     pr_info,
                     dump_to_file=True,
                 )
+                if job_report.status != SUCCESS:
+                    exit_code = 1
```

Comment:
why do you need this?
I don't see a reason for making POST step red, RUN step is red already in this case

Reply:
the case is that the Run is not. I'll take a look there, as we discussed